### DISCUSSION TITLE
avatar not found

### DISCUSSION THREAD

**Discussion by glagioia** [faker-js/faker#122] [Q&A] [answered]
hello people 👍 , does anyone have a problem not loading avatars?

**Comment by Shinigami92** [comment, marked as answer]
6.0.0-alpha is not released yet 
See #28 for more

**Reply by glagioia** [reply]
Oh I see, I am using the tool to practice as I am new to this and thought maybe I was doing something else. For now replace all avatar photos with the same one. Thanks!
